docs/docs/20-admins/30-operation/100-global-environments.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Global Environments
+---
+
+Renku administrators have the ability to create global environments that are
+accessible by any user and can be used in any session launcher.
+
+:::note Image visibility
+The Docker images used for global environments have to be public.
+:::
+
+## Creating a Global Environment
+
+1. Sign in as a Renku administrator
+2. Navigate to the Renku administrator panel, you should have a link to it in
+the menu that shows up when you click on your profile icon in the upper right
+corner of the Renku dashboard.
+3. Click on "Add Session Environment".
+4. Enter the docker image you want to use for the environment, the name and description.
+5. If you are using a Renku image as your base then you need to modify the following options
+  in the advanced settings section:
+    - Default URL: `/` (if this is a Jupyterlab image this should be `/lab`)
```

Comment:
NOTE: the current version of JupyterLab redirects `/` to `/lab`, so it is not necessary to use `/lab`.